dashboard/src/config/webpack.app.js

```diff
@@ -57,19 +64,22 @@ var config = getConfig({
   }
 })
 
-// Enable babel-polyfill
-config.entry = ['babel-polyfill', './src/app.js']
-
-// Enable CSS modules
+// Customize loader configuration
 let loaders = config.module.loaders
 
 for (let item of loaders) {
-  if (item.loader) {
-    item.loader = item.loader.replace('css-loader','css-loader?modules&importLoaders=1&localIdentName=[name]__[local]__[hash:base64:5]')
+  // Enable CSS module support
+  if (item.loader && item.loader.indexOf('css-loader') > 0) {
```

Comment:
If we made this an `if/else if/else` ladder it would clarify that these are mutually exclusive cases.